packages/bpk-component-autosuggest/src/BpkAutosuggest.js

```diff
@@ -49,10 +51,16 @@ Autosuggest.defaultProps.theme = {
   sectionTitle: getClassName('bpk-autosuggest__section-title'),
 };
 
-Autosuggest.defaultProps.renderInputComponent = inputProps => {
+type Props = {
+  ref: (?HTMLInputElement) => mixed | ?Function,
+  inputRef: (?HTMLInputElement) => mixed | ?Function,
```

Comment:
Out of interest, do we need the `?Function` part or will just `mixed` work? I figured `mixed` would include `Function`

Reply:
I think because in the useage below we check for a function and I thought that was right over a HtmlInputElement

Reply:
Changed and seems to be fine